def throw(ctx, message):
    raise Exception(message)
